modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Tags/TagDefinitionDictionary.cs

```diff
@@ -0,0 +1,20 @@
+﻿using JetBrains.Annotations;
+using System.Collections.Generic;
+using Volo.Abp.Localization;
+
+namespace Volo.CmsKit.Tags
+{
+    public class TagDefinitionDictionary : Dictionary<string, TagDefiniton>
+    {
+        public void AddOrReplace(
```

Comment:
There is a method called `AddIfNotExists` inside `AbpCollectionExtensions`. We can use that for adding definitions. I don't see any use case for replacing definitions. Am I missing anything?

Reply:
The pattern comes from here: https://github.com/abpframework/abp/blob/a92a5c98ab95aab4c456d9068c141e6fddba8109/modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Reactions/ReactionDefinitionDictionary.cs#L9